packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -333,8 +333,9 @@ Future<String> _buildAotSnapshot(
       sdkRoot: artifacts.getArtifactPath(Artifact.flutterPatchedSdkPath),
       mainPath: mainPath,
       extraFrontEndOptions: extraFrontEndOptions,
-      linkPlatformKernelIn : true,
-      aot : true,
+      linkPlatformKernelIn: true,
+      aot: true,
+      trackWidgetCreationLocations: false,
```

Comment:
Here and below - why do you need to pass explicit default `false` value for trackWidgetCreationLocations? It seems that quite few call-sites in this PR don't really have to be changed.

Reply:
don't need to. I tracked down all the call sites to make sure I wasn't missing ones I cared about but I can revert the ones that are not needed.